styles/components/Navbar.scss

```diff
@@ -3,199 +3,207 @@
+          position: absolute;
+          transition: all 0.5s ease-in-out;
+          width: 100%;
+        }
+
+        &::before {
+          transform: translateY(8px);
+        }
+
+        &::after {
+          transform: translateY(-8px);
+        }
       }
-    }
-  }
 
-  #menu-toggle {
-    display: none;
-  }
+      &.active {
+        .bar {
+          background-color: transparent;
+          transform: translateX(-50px);
+        }
 
-  #hamburger-menu {
-    background: rgba(255, 255, 255, 0.95);
-    bottom: 0;
-    display: flex;
-    flex-direction: column;
-    height: 100%;
-    justify-content: center;
-    left: 0;
-    position: fixed;
-    top: 0;
-    transform: translate3d(0, -100%, 0);
-    transition: transform 0.35s cubic-bezier(0.05, 1.04, 0.72, 0.98);
-    width: 100%;
-    z-index: -20;
-
-    ul.nav-items {
-      display: flex;
-      flex-direction: column;
-      float: none;
-      justify-content: center;
-      list-style: none;
-      margin: auto;
-      padding: 0;
-      text-align: center;
+        .bar::before {
+          transform: rotate(-45deg) translate(35px, 35px);
+        }
+
+        .bar::after {
+          transform: rotate(45deg) translate(35px, -35px);
+        }
+      }
+
+      &:hover {
+        // which color to keep?
+        // border-color: $font-primary-color;
+        border-color: $acm-cobalt !important;
+
+        .bar,
+        .bar::before,
+        .bar::after {
+          // background-color: $font-primary-color;
+          background-color: $acm-cobalt !important;
+        }
+      }
     }
 
-    ul.nav-items li {
-      display: block;
-      font-size: 1.8em;
-      font-weight: 600;
-      margin: 40px auto;
-      opacity: 0;
+    .nav-items {
+      align-items: center;
+      display: flex; // will become none at smaller screen size
+      list-style: none;
+      margin: 0;
       padding: 0;
-      text-transform: lowercase;
-      transform: translate3d(0, -50%, 0);
-      transition: all 1.25s cubic-bezier(0.05, 1.04, 0.72, 0.98);
-    }
 
-    ul.nav-items li:hover {
-      color: #999;
-      cursor: pointer;
+      li {
+        font-size: 0.95em;
+        font-weight: 600;
+        height: 40px;
+        line-height: 40px;
+        padding: 0 20px;
+        text-transform: lowercase;
+        transition: 0.2s;
+
+        a,
+        a:visited {
+          color: inherit;
+          text-decoration: none;
+
+          &.active {
+            color: $acm-cobalt !important;
+          }
+        }
+        
+        &:hover {
+          // which color to keep?
+          // color: #999;
+          color: $acm-cobalt !important;
+          cursor: pointer;
+        }
+      }
 
+      #committee-mobile-nav-item {
+        display: none;
+        height: max-content;
+        text-align: center;
+        width: 60%;
 
-    }
+        &:hover {
+          // prevents all committees from highlighting
+          color: $font-primary-color !important;
```

Comment:
I think it would also be super cool if you could make each committee's hover color that committee's primary color (i.e. hack is purple, ai is light blue). Might be out of scope of this PR, but a good followup issue if it's not done here!